modules/ROOT/pages/gov-cloud-load-balancer.adoc

```diff
@@ -0,0 +1,100 @@
+endif::[]
+
+A Government Cloud dedicated load balancer enables you to route external HTTP and HTTPS traffic to multiple 
+Mule applications deployed into Government Cloud workers inside a Virtual Private Cloud (VPC). Each Government Cloud
+dedicated load balancer is assigned to an Anypoint VPC, and that Anypoint VPC runs within a particular service region. 
+The dedicated load balancer routes both external and VPC-internal traffic to Government Cloud workers deployed into that 
+Anypoint VPC.
+
+The Government Cloud load balancer already redirects these requests, where `_myApp_` is the name of the Mule 
+application deployment to Government Cloud:
+
+* HTTP requests to `+http://myApp.cloudhub.io+` redirects to `mule-worker-myApp.cloudhub.io:8081`
+* HTTPS traffic to `+https://myApp.cloudhub.io:443+` redirects to `+https://mule-worker-myApp.cloudhub.io:8082+`
+
+Your load balancer listens for outside requests over HTTPS and, by default, communicates internally with your 
+worker over HTTP. If you configured your Mule application within the Anypoint VPC to listen on HTTPS, make sure 
+you set `upstreamProtocol` to `HTTPS` when creating the mapping list using the 
+xref:anypoint-platform-cli-commands.adoc#cloudhub-load-balancer-mappings-add[`load-balancer mappings add`] command.
+
+
+== Routing External Requests to the Dedicated Load Balancer
+
+A Government Cloud dedicated load balancer provides an alternative domain name to route HTTP requests to 
+Mule applications listening on port 8091 and HTTPS requests to Mule applications listening on port 8092. You can 
+also use mapping rules to rename requests to the Government Cloud dedicated load balancer to a different Mule 
+application domain name.
+
+The Government Cloud dedicated load balancer exposes an external domain name that resolves to two public IP 
+addresses which are accessible from outside your Government Cloud Anypoint VPC network. `_lb-name_` is the name you 
+gave the load balancer when you created it:
+
+----
+ _lb-name.lb-gprod-rt.anypointdns.net
+----
+
+== Renaming External Mule Application URLs Through a Dedicated Load Balancer
+
+You can create a mapping rule to rename the Mule application, so it can be accessed through a different domain name.
+
+For example, suppose you deploy a Mule application named myApp-myCompany-prod.cloudhub.io. Government Cloud domain names 
+must be globally unique among every other MuleSoft Government Cloud customer and Mule application. A dedicated load balancer helps 
+you hide this naming complexity within your corporate DNS domain name.
+
+You can set a mapping rule in your dedicated load balancer so external clients can access the Mule application 
+on `+http://myApp.lb-name.lb-gprod-rt.anypointdns.net+`, or `+https://myApp.lb-name.lb-gprod-rt.anypointdns.net:443+`.
+
+In addition, you can set a CNAME record in your corporate DNS nameserver to mask the `lb-name.lb-gprod-rt.anypointdns.net` 
+domain to your company's "vanity" domain.
```

Comment:
Will your audience be familiar with CNAME?

```suggestion
In addition, you can set a CNAME record in your corporate DNS nameserver to mask the `lb-name.lb-gprod-rt.anypointdns.net` domain by using instead your company's "vanity" domain.
```
And will they know what a "vanity" domain is?